style.css

```diff
@@ -0,0 +1,143 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&family=Roboto:wght@500&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  height: 718px;
+  width: 375px;
+  background: url('./images/background1.png');
+  display: flex;
+  flex-direction: column;
+  background-size: cover;
+  background-repeat: no-repeat;
+  left: 0;
+  top: 0;
+  background-position: center;
+}
+
+.main-bar {
+  height: 20%;
+  width: 100%;
+  background-color: brown;
+  display: flex;
+}
+
+.main-content {
+  height: 80%;
+  width: 100%;
+  display: flex;
+}
+
+.main-bar .right {
+  height: 100%;
+  width: 50%;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  gap: 30px;
+  align-items: flex-end;
+}
+
+.main-bar .left {
+  height: 100%;
+  width: 50%;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  padding: 20px;
+  gap: 30px;
+}
+
+.container {
+  position: relative;
+  top: 110px;
+  left: 50px;
+  height: 75%;
+  width: 75%;
+}
```

Comment:
- [ ] Please align the contents of this container with flexbox. This gives us a lot of options to align all our elements, plus, it's responsive. You can read more about flexbox and how to use it [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)